phpunit.xml.dist

```diff
@@ -18,4 +11,9 @@
       <group>disable</group>
     </exclude>
   </groups>
+  <source>
```

Comment:
@laminas/technical-steering-committee any ideas (besides "do not run tests on latest") to somehow have tests not failing on "latest" when phpunit changes config? No clue why sebastian is dropping still compatible (but deprectated) configuration flags entirely even tho they still work...
So config was valid, even for the "latest" version but as per the schema, it was not.
Should I ask in phpunit why there are config flags dropped from the schema instead of marking them deprecated? Does XSD even support the concept of "deprecated"?